R/utils.R

```diff
@@ -253,3 +253,8 @@ isAvailable <- function(package, min_version = NULL) {
   # the package has at least that version.
   return(utils::packageVersion(package) >= min_version)
 }
+
+# similar to rex::escape
```

Comment:
Maybe this comment would be better if it described what `escapeRegex` does in a sentence?